src/python/pants/backend/python/python_artifact.py

```diff
@@ -1,64 +1,43 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+from typing import Any, Dict
+
 
 class PythonArtifact:
     """Represents a Python setup.py-based project."""
 
-    class MissingArgument(Exception):
```

Comment:
Most of the validation moved from here to `SetupKwargs` in `run_setup_py.py`, so that we account for any values added via plugins.